src/sentry/api/endpoints/api_token_details.py

```diff
@@ -0,0 +1,92 @@
+from django.utils.decorators import method_decorator
+from django.views.decorators.cache import never_cache
+from rest_framework import serializers
+from rest_framework.fields import CharField
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import analytics
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import Endpoint, control_silo_endpoint
+from sentry.api.endpoints.api_tokens import _get_appropriate_user_id
```

Comment:
prepended `_`methods should _only_ be used within the files they're defined.
If you're making this a global helper, then we should remove the `_`

Reply:
good catch!